src/ResourceManager/Sql/Commands.Sql/Commands.Sql.csproj

```diff
@@ -447,10 +452,15 @@
       <Project>{04B53727-16E6-4EF0-9116-04C7F1C33842}</Project>
       <Name>SqlManagement</Name>
     </ProjectReference>
+    <Reference Include="Microsoft.Azure.KeyVault.Core, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
you can remove this, it is a repeat from common.dependencies.

Reply:
Removed.